server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk

```diff
@@ -0,0 +1,353 @@
+            },
+            items: [
+              {
+                label: "Day",
+                name: "validToDateDay",
+                classes: "govuk-input--width-2",
+                autocomplete: "off",
+                value: formValues.validToDateDay
+              },
+              {
+                label: "Month",
+                name: "validToDateMonth",
+                classes: "govuk-input--width-2",
+                autocomplete: "off",
+                value: formValues.validToDateMonth
+              },
+              {
+                label: "Year",
+                name: "validToDateYear",
+                classes: "govuk-input--width-4",
+                autocomplete: "off",
+                value: formValues.validToDateYear
+              }
+            ]
+          }) }}
+        {% endset -%}
+        {{ govukCheckboxes({
+          name: "hasEndDate",
+          fieldset: {
+            legend: {
+              text: "Does this session have an end date?",
+              classes: "govuk-fieldset__legend--m"
+            }
+          },
+          errorMessage: errors | findError('hasEndDate'),
+          items: [
+            {
+              text: "Yes",
+              value: "yes",
+              conditional: {
+              html: endDateHtml
+            },
+              checked: formValues.hasEndDate === "yes"
+            }
+          ]
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Open capacity",
+            classes: "govuk-label--m"
+          },
+          hint: {
+            text: "What is the open capacity?"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "openCapacity",
+          name: "openCapacity",
+          inputmode: "numeric",
+          value: formValues.openCapacity,
+          autocomplete: "off",
+          errorMessage: errors | findError('openCapacity')
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Closed capacity",
+            classes: "govuk-label--m"
+          },
+          hint: {
+            text: "What is the closed capacity?"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "closedCapacity",
+          name: "closedCapacity",
+          inputmode: "numeric",
+          value: formValues.closedCapacity,
+          autocomplete: "off",
+          errorMessage: errors | findError('closedCapacity')
+        }) }}
+
+        {{ govukInput({
+          label: {
+            text: "Visit room",
+            classes: "govuk-label--m"
+          },
+          hint: {
+            text: "What is the visit room name?"
+          },
+          classes: "govuk-!-width-one-third",
+          id: "visitRoom",
+          name: "visitRoom",
+          value: formValues.visitRoom,
+          autocomplete: "off",
+          errorMessage: errors | findError('visitRoom')
+        }) }}
+
+        {# <h2 class="govuk-heading-m">Category groups</h2>
```

Comment:
Does this block want to be here if it's commented out?

Reply:
made the decision to leave it in as spent a fair bit of time working the logic out between here and the route file - only for us to decide not to use it, so will mean we are more prepared when we do look to include this

Reply:
This will be put in shortly so all good to leave for now.